activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Introduce new database configuration options `keepalive`, `max_age`, and
+    `min_connections` -- and rename `pool` to `max_connections` to match.
```

Comment:
I like the `min_connections` and `max_connections` rename to help clarify that "max connections" means "up to" the number would be in use when activity is high.